src/main/java/net/fabricmc/loader/impl/util/SystemProperties.java

```diff
@@ -34,6 +34,8 @@ public final class SystemProperties {
 	public static final String LOG_LEVEL = "fabric.log.level";
 	// additional mods to load (path separator separated paths, @ prefix for meta-file with each line referencing an actual file)
 	public static final String ADD_MODS = "fabric.addMods";
+	// a comma-separated list of mod ids to disable, even if they're discovered. mostly useful for unit testing.
+	public static final String DISABLE_MOD_IDS = "fabric.disableModIds";
```

Comment:
```suggestion
	public static final String DISABLE_MOD_IDS = "fabric.debug.disableModIds";
```

I think this would be better noted as a debug option similar to `fabric.debug.loadLate`

Reply:
Yeah, I’m happy with that. My main worry was people using it for launchers or whatever to disable mods when this option is really a last resort. Ideally you should still be totally removing the mod.